src/Features/Core/Portable/Diagnostics/EngineV2/DiagnosticIncrementalAnalyzer.CompilationManager.cs

```diff
@@ -118,7 +118,7 @@ public Task<CompilationWithAnalyzers> CreateAnalyzerDriverAsync(Project project,
                 // in IDE, we always set concurrentAnalysis == false otherwise, we can get into thread starvation due to
                 // async being used with syncronous blocking concurrency.
                 return new CompilationWithAnalyzersOptions(
-                    options: new WorkspaceAnalyzerOptions(project.AnalyzerOptions, project.Solution.Workspace),
+                    options: new WorkspaceAnalyzerOptions(project.AnalyzerOptions, project.Solution.Options, project.Solution.Workspace),
```

Comment:
are you asking me or @jasonmalinowski ? I didn't change this semantic. also, I am not sure whether in current design where analyzer lives in compiler layer there is a way to pass in solution along with analyzer bits to compiler.